### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by rusty-snake** [NeoApplications/Neo-Store#573] [Q&A]
- Last commit 2 month ago
- Last release over 1 year ago
- Outstanding fixes to be released for nearly 1 year

==> What's the project status?

**Comment by machiav3lli** [comment]
The project is lively as ever (last commit on a dev branch is 2 weeks ago. last test release 2 months ago. the one year release cycle is related to developing v1.0.0, I wouldn't consider that unusual. Plus, I'm not aware of any major "outstanding fixes".), it's just that work, academics and private life had slowed things down.
